### PR TITLE
Fixed JSON issue

### PR DESCRIPTION
**Pull request by dlucci**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -73,6 +73,13 @@
 			<artifactId>commons-io</artifactId>
 			<version>2.1</version>
 		</dependency>
+		
+		<dependency>
+			<groupId>commons-codec</groupId>
+			<artifactId>commons-codec</artifactId>
+			<version>1.9</version>
+		</dependency>
```
Comment (dpilipenko) [inline pom.xml]: Do not want.

File: build.gradle [hunk 2]
```diff
@@ -1,6 +1,7 @@
 buildscript {
     repositories {
         mavenCentral()
+	mavenLocal()
```
Comment (dpilipenko) [inline build.gradle]: Do we want this? Is this so I don't download all dependencies each time I do a clean build?

File: src/main/java/skunkworks/matrix/jive/responses/EmailResponse.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import java.util.List;
+
+public class EmailResponse {
+	public String id;
+	public String[] tags;
+	public String displayName;
+	public List<Email> emails;
+	public List<Photos> photos;
+
```
Comment (dpilipenko) [inline src/main/java/skunkworks/matrix/jive/responses/EmailResponse.java]: Wut!? you can do this? V i thought each class needs to be in it's own .java file. #neat

File: build.gradle (hunk 2 above)
Comment (dlucci) [inline build.gradle]: yes.  that will tell the build to look in  the local maven repo first before trying to download a dependency

File: pom.xml (hunk 1 above)
Comment (dlucci) [inline pom.xml]: my code cannot build without it.  it is either that or upgrade to java 8

File: src/main/java/skunkworks/matrix/jive/responses/EmailResponse.java (hunk 3 above)
Comment (dlucci) [inline src/main/java/skunkworks/matrix/jive/responses/EmailResponse.java]: (boom)

File: pom.xml (hunk 1 above)
Comment (dpilipenko) [inline pom.xml]: oh that's right  - i forgot

**Comment by dpilipenko** [comment]
Consulted with management. Approved.

**Pull request merged**